### DISCUSSION TITLE
Have you a working `CustomRule` - example?

### DISCUSSION THREAD

**Discussion by lava83** [spatie/laravel-data#263] [Q&A]
Have you? What will be implemented in `CustomRule` - which will be returns in the `getRules` - array like here:

```php
public function getRules(): array
{
    return [new CustomRule()];
}
```

**Comment by rubenvanassche** [comment]
Take a look at our validation attributes here: https://github.com/spatie/laravel-data/tree/main/src/Attributes/Validation

They basically all are implemented like a custom rule.

`getRules` should return the rules like you would write them down in Laravel.
